R/transformCounts.R

```diff
@@ -228,6 +235,7 @@ setMethod("transformAssay", signature = c(x = "SummarizedExperiment"),
     FUN <- switch(method,
                   log10 = .calc_log,
                    log2 = .calc_log,
+                  css = .calc_css,
```

Comment:
Can you make the indentation to e multiplication of 4 spaces (also the lines 236-237)